tests/integration/datastore/test_datastore.py

```diff
@@ -14,10 +14,52 @@
 You should have received a copy of the GNU Affero General Public License
 along with nucypher.  If not, see <https://www.gnu.org/licenses/>.
 """
+import msgpack
 import pytest
 from datetime import datetime
 
 from nucypher.datastore import datastore, keypairs
+from nucypher.datastore.db.models import DatastoreRecord, RecordField
+
+
+def test_datastore_record():
+
+    class TestRecord(DatastoreRecord):
+        test = RecordField(bytes)
+        test_date = RecordField(datetime,
+                encode=lambda val: datetime.isoformat(val).encode(),
```

Comment:
Why a lambda?

Reply:
The `DatastoreRecord`, when encoding or decoding, passes the value as an arg into the callable given. In this case, I need to make it a lambda so that it can encode the string into a utf-8 bytestring.